cmd/scollector/collectors/processes_windows.go

```diff
@@ -10,31 +10,28 @@ import (
 	"bosun.org/opentsdb"
 )
 
-var regexes = []*regexp.Regexp{}
+var regexesProcesses = []*regexp.Regexp{}
 
 func AddProcessConfig(line string) error {
 	reg, err := regexp.Compile(line)
 	if err != nil {
 		return err
 	}
-	regexes = append(regexes, reg)
+	regexesProcesses = append(regexesProcesses, reg)
 	return nil
 }
 
 func WatchProcesses() {
-	if len(regexes) == 0 {
+	if len(regexesProcesses) == 0 {
 		// if no processes configured in config file, use this set instead.
-		regexes = append(regexes, regexp.MustCompile("chrome|powershell|scollector|SocketServer|WinRM|MSSQLSERVER|StackServerProd|StackServerDev|LogStasher"))
+		regexesProcesses = append(regexesProcesses, regexp.MustCompile("chrome|powershell|scollector|SocketServer|WinRM|MSSQLSERVER|StackServerProd|StackServerDev|LogStasher"))
```

Comment:
That is is scheduled for after the PR is merged and conf files are deployed. Once we are working as expected on conf files I'll update the default settings to remove anything from stackexchange.